frontend/app/(defaultLayout)/Troubleshooting/[device]/components/Answers.tsx

```diff
@@ -17,7 +17,7 @@ export function Answers({
                <Heading
                   as={hasProblems ? 'h4' : 'h3'}
                   color="gray.800"
-                  fontSize={{ base: '24px', sm: '30px' }}
+                  fontSize={{ base: '20px', mdPlus: '24px' }}
```

Comment:
Is `mdPlus` a custom breakpoint we created? I haven't seen it before

Reply:
Yeah, I added it here: https://github.com/iFixit/react-commerce/pull/2143/files#diff-9e3cebe6686b9c7a2c8f51158b777b7f8d7b7bd53fd8a126776adc1e655bf956R7

tl;dr: Chakra uses min-width @media queries, so we need 'md' + 1px